### DISCUSSION TITLE
How to remove the MSIX project?

### DISCUSSION THREAD

**Discussion by pellea** [microsoft/WindowsAppSDK#1147] [Q&A]
I tried to follow the steps here https://nicksnettravels.builttoroam.com/no-packaging-project/ to remove the MSIX project.
But I still can't make it works.

Any tips or samples to get rid of it?

I wish I could get access to the hot reload feature of VS2022.

(TargetFramework: net6.0)

**Comment by ghost1372** [comment]
mybe net6.0 is not supported, try with net5.0
